### ISSUE TITLE
Decentralists party and AtomOne hub have different governing bodies.

### ISSUE THREAD

**Issue by jaekwon** [atomone-hub/genesis#7]
The Decentralists is a party of equal individuals.
The AtomOne hub is a proof-of-stake hub based on $ATOM1.
They are related but different.
It's almost like one is the democratic view of the other.
How many $ATOM1 does one have to own to be a Decentralist? 
I suppose it doesn't matter as long as you can prove your worth on chain e.g. even through contributions and discussions. But Decentralists I suppose can be expected to have at least one. 

The main thing is, they are different governing bodies, so this should be reflected in all aspects of the document.

**Comment by moul** [comment]
How about ensuring that Decentralists never control more than 50% of $ATOM1 shares and voting power (perhaps even less, like 1/3)? This way, the proof-of-stake hub will always be predominantly governed by its end-users.

**Comment by jaekwon** [comment]
I like it except this part: "How about ensuring that Decentralists never control more than 50% of $ATOM1 shares and voting power, and possibly even less, like 1/3?"; I think the Decentralists are in theory up to 100% aligned with $ATOM1 in the beginning, and it can drift over time naturally, and some precautions must be taken as a result.

**Comment by moul** [comment]
I'm all for the alignment we're aiming at (100%). Yet, I believe this proof-of-stake hub could be stronger when its stakers have a say, like voting rights. Maybe we can start at 100% and then gradually share more control with the stakers. How about aiming for a 50% share in a few years?

I think everyone who's really into building good stuff will want to be part of Decentralists. However, I feel it's not as effective in spreading control as Gno's proof-of-contribution and GovDAO. But, by combining it with the stakers, we get the best of both worlds: lots of people involved and high-quality contributions, covering both builders and users.

**Comment by jaekwon** [comment]
I updated the README with a form of mild integration into governance.
Lots to iron out about this.

**Comment by jaekwon** [comment]
I split this out entirely so ATOM ONE doesn't include the Decentralists much at all except once by reference.
And this seems right, because the Decentralists should be its own independent project. They complement one another but they also should not mix together excessively so as to allow for independent development. This is more decentralized thank linking them too early too soon forcing us to figure everything out at once.

**Comment by drewstaylor** [comment]
What so you mean by "the Decentralists"? You seem to be talking about something quite specific, not just people who prefer decentralization

**Comment by OrionUnix** [comment]
Above all, only people who have had an ATOME ONE for a certain length of time should be able to vote, and voting power should not be linked to the number of ATOME ONEs you have.
I mean, we'd have to make sure that voting is done under fairly strict conditions, to avoid any one person or group putting forward a proposal that could harm the ecosystem.

**Comment by jaekwon** [comment]
If we do this then we make it difficult for ATOM1/ATOME/ATMO holders to make necessary transfers (e.g. wallet migrations) without unnecessarily punishing them.

For the purpose of this experiment we should make $ATOM1 *the* staking token, and this means it SHOULD be linked to the number of tokens you have. 

For non-stake-based weighting I would look at the Decentralists.md file, and github.com/decentralists/ and consider creating more structure and contributing to the disucussions, but creating a democratically weighted body creates a new governing body, and this raises a LOT of new questions, and I believe trying to figure out all of the details of the Decentralists is a tall order (because we NEED to get this constitution mostly perfect before laucnhing) and therefore while we can begin to fund such an initiative (with a lot of tooling and UX etc to go with it; maybe it is several open source projects interwoven together like GMail and GoogleDocs and GoogleSheets etc; and maybe integration between these two distinct bodies in governance should be done as child-chains generated from AtomOne; and we can have a variety of experiments that blend these approaches.

But as you can see from the github history, before I did try to integrate the Decentralists in, but this made the main document and the model way too complex. It is worth minimzing the objective for the purpose of AtomOne, and this is also more aligned with the concept of the minimal hub. 

tldr; let's keep it simple if possible.

**Comment by 0xFDg** [comment]
For my point of view we need a filter into the governance. Is it right for the people to vote on the future of the chain ? Or better, is it right that someone without knowledge vote and decide the future of the chain without data, knowledge or any other competence ? For me w can do only 2 thing:

Decentralists : 

1. They are entities that predict the future of the chain, imagine someone in front of 100 streets making decisions blindfolded, it's crazy. Decentralist have the role to select a shortlist of 3-4 best roads, illuminating them and the hub-users chose one of this 3.
2. They are people with skills, how? By passing an exam, we create an exam with technical questions about the chain with topic on Defi finance, economy and ATOM1 technology, if you pass the exam and if you have at least 1 ATOM1 in stake, you are one of the Decentralists.

For me other solutions have not sense.

**Comment by 0xFDg** [comment]
And also Decentralist can be divide in sub area with 2-3 menager for each area.

**Comment by giunatale** [comment]
> The Decentralists is a party of equal individuals.
The AtomOne hub is a proof-of-stake hub based on $ATOM1.
They are related but different.

and

> And this seems right, because the Decentralists should be its own independent project. They complement one another but they also should not mix together excessively so as to allow for independent development.

From Jae pretty much encapsulates my feelings as well.
I don't think we want to "institutionalize" the Decentralists in AtomOne.

But there are interesting points about governance here, maybe we want to carry them over in #40
